### PR TITLE
First degree indicators

### PR DESCRIPTION
**Pull request by heron2014**
# Ready to merge

Related: 
[issue 67](https://github.com/FAC-GM/app/issues/67)
[issue 70](https://github.com/FAC-GM/app/issues/70)

UI at the moment : 
initials

### CODE REVIEW

**Comment by nelsonic** [comment]
@heron2014 looks good.
@iteles please second-pair-of-eyes :eyes:  & merge. thanks.

**Comment by iteles** [comment]
@heron2014 Could you add a 'Functionality' section to the readme and ensure a one-sentence explanation of this feature is included as a bullet point please?

**Comment by iteles** [comment]
Minor UI bug:
screen shot 2015-11-24 at 15 38 05

File: assets/css/mobile-first.css [hunk 1]
```diff
@@ -129,6 +129,15 @@ a#mobile {
       color: grey;
      }
 
+     .headline ul li span.initials {
```
Comment (iteles) [inline assets/css/mobile-first.css]: Why do we need all of these selectors? Could we just use `.initials`?

or have the search class be `search-initials` and the main profile be `profile-initials`?

I'm a big fan of [selector intent](http://csswizardry.com/2012/07/shoot-to-kill-css-selector-intent/)

File: lib/home.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ exports.register = function (server, options, next) {
           index: process.env.ES_INDEX,
           from: (pageNum - 1) * perPage,
           size: perPage,
-          fields: ['id', 'picture','fullname', 'current', 'location'],
+          fields: ['id', 'picture','fullname', 'current', 'location', 'connectedWith'],
```
Comment (iteles) [inline lib/home.js]: I would suggest that in the interest of it being really obvious what this is, I would call this field `connectedTo` (from an English perspective, it's a little more obvious for the developers coming after us)

File: views/helpers/initialsConnectedUsersHelper.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = function(name) {
+
+  return name.match(/\b(\w)/g).join('');
```
Comment (iteles) [inline views/helpers/initialsConnectedUsersHelper.js]: @SimonLab @heron2014 `\b` is a 'bad part' of JS according to Crockford :blush: (last sentence in [this paragraph](https://github.com/iteles/Javascript-the-Good-Parts-notes#regexp-escape) of notes) - could you spend 15 minutes investigating if there's a better way of doing this please?

**Comment by heron2014** [comment]
@iteles good spot! Fixing now

**Comment by iteles** [comment]
@heron2014 Thanks!

**Comment by iteles** [comment]
screen shot 2015-11-24 at 16 39 11

**Comment by heron2014** [comment]
:+1: thanks @iteles

**Pull request merged**